contrib/builder/rpm/generate.sh

```diff
@@ -57,7 +61,11 @@ for version in "${versions[@]}"; do
 		sqlite-devel # for "sqlite3.h"
 		tar # older versions of dev-tools don't have tar
 	)
-	echo "RUN yum install -y ${packages[*]}" >> "$version/Dockerfile"
+	if [ "$from" == "oraclelinux:7" ]; then
+		echo "RUN yum install --enablerepo=ol7_optional_latest -y ${packages[*]}" >> "$version/Dockerfile"
```

Comment:
Ooh, that sounds like a reasonable solution!

Something like this would probably be enough, really:
```bash
case "$from" in
    oraclelinux:7)
        packages=( --enablerepo=ol7_optional_latest "${packages[@]}" )
        ;;
esac
```